tools/check

```diff
@@ -241,6 +241,21 @@ should_run_build_runner() {
     return 1
 }
 
+run_l10n() {
```

Comment:
nit: move this above `should_run_build_runner`, which is a helper function specific to `run_build_runner`, so as not to split those from each other

(alternatively, move this suite after the `build_runner` suite)